package/base-files/files/lib/functions/system.sh

```diff
@@ -115,6 +115,12 @@ macaddr_add() {
 	echo $oui:$nic
 }
 
+macaddr_geteui() {
+	local mac=$1
+	local sep=$2
+	echo ${mac:9:2}$sep${mac:12:2}$sep${mac:15:2}
+}
```

Comment:
Maybe add an empty line between the variable assignments and the echo command (as in my example). This will match the style of the other functions in the file.